docs/source/gbd2017_models/causes/neoplasms/stomach_cancer/cancer_model.rst

```diff
@@ -0,0 +1,456 @@
+
+   * - Restriction Type
+     - Value
+     - Notes
+   * - Male only
+     - False
+     -
+   * - Female only
+     - False
+     -
+   * - YLL only
+     - False
+     -
+   * - YLD only
+     - False
+     -
+   * - YLL age group start
+     - age_group_id = 8
+     - 15-19 years
+   * - YLL age group end
+     - age_group_id = 235
+     - 95+ years
+   * - YLD age group start
+     - age_group_id = 8
+     - 15-19 years
+   * - YLD age group end
+     - age_group_id = 235
+     - 95+ years
+
+Vivarium Modeling Strategy
+--------------------------
+
+Scope
++++++
+
+This Vivarium modeling strategy is intended to simulate stomach cancer incidence/morbidity as well as mortality so that it reflects the estimates and assumptions of GBD. Additionally, this cause model intends to allow for the differentiation of preclinical screen-detectable (asymptomatic) phase of stomach cancer and the clinical (symptomatic) phase of stomach cancer. 
+
+Assumptions and Limitations
++++++++++++++++++++++++++++
+
+1. This model will assume the existence of a "recovered" cause model state in an attempt to be consistent with the GBD assumption that no morbidity due to stomach cancer occurs more than ten years past incidence of the *clinical* phase of stomach cancer. The assumption also asserts that there is no recurrance of stomach cancer.
+
+2. This model assumes that the GBD incidence rate corresponds to the incidence of asymptomatic stomach cancer rather than *clinically detected* stomach cancer arising from symptomatic presentation at the doctor's office, which is a mix of pre-clinical (detection of pre-clinical stomach cancer from other non-stomach cancer related reasons) and clinical detections. This assumption has a few notable downstream limitations, including:
+
+	- simulation incidence of *clinical* stomach cancer will lag slightly behind forecasted incidence of stomach cancer due to the mean sojourn time period delay
+  - assume a short mean sojourn time 
+
+.. todo::
+
+  think more about these assumptions in relation to the sojourn time
+
+3. The prevalence of preclinical/screen-detectable stomach cancer is assumed to be equal to prevalence of detected stomach cancer (GBD prevalence of stomach cancer) scaled to the ratio of duration spent in the preclinical/screen-detectable state (mean sojourn time) and the clinical state (average survival time). This method relies on the assumption that GBD prevalence of stomach cancer represents clinical stomach cancers; this may be a reasonable assumption for China given that they do not have an aggressive screening program. 
+
+
+Cause Model Diagram
++++++++++++++++++++
+
+.. image:: cause_model_diagram.svg
+
+State and Transition Data Tables
+++++++++++++++++++++++++++++++++
+
+.. list-table:: State Definitions
+   :widths: 5 5 20
+   :header-rows: 1
+
+   * - State
+     - State Name
+     - Definition
+   * - S
+     - Susceptible
+     - Without cancer condition (may have pre-cancer states)
```

Comment:
perhaps note that pre-cancer states could be implemented with a Risk Exposure, and it is intentional that it is not included in this cause model document.

Reply:
yes- will add

Reply:
will also update the the 'state machine diagram' (James labelled them as such) in the concept model doc.